lib/plugin/playwrightCoverage.js

```diff
@@ -0,0 +1,36 @@
+const coverage = require('../coverage');
+
+/**
+ * Dumps playwright code coverage after every test.
+ *
+ * #### Configuration
+ *
+ * Configuration can either be taken from a corresponding helper (deprecated) or a from plugin config (recommended).
```

Comment:
```suggestion
```

This line is confusing. Better not to start with something that has an alternative on first line. 
If the heper's config is deprecated let's not mention it at all